tests/graph/test-data/rust/src/seeding/distributions/ontology/shortname.rs

```diff
@@ -14,23 +14,23 @@ use crate::seeding::distributions::{
 };
 
 #[derive(Debug, Clone, PartialEq)]
-enum InnerShortnameDistribution<'s> {
-    Const(ConstDistribution<Cow<'s, str>>),
-    Weighted(WeightedDistribution<ConstDistribution<Cow<'s, str>>>),
-    Uniform(UniformDistribution<ConstDistribution<Cow<'s, str>>>),
+enum InnerShortnameDistribution {
```

Comment:
Wouldn't it make sense here to use something like https://docs.rs/fake/latest/fake/ to generate genuine usernames / shortnames?

Reply:
Have not looked at name generators, yet, but yes, a proper distribution for names would be helpful. Although, for short names I used the local ID ans shard ID to make sure they are really unique by design.

Reply:
Yep, just wanted to flag it 👍